src/components/Search/index.tsx

```diff
@@ -53,7 +53,7 @@ const Search: React.FC = () => {
   };
 
   if (error) {
-    return <div>{error}</div>;
+    console.error(error);
```

Comment:
Is it intentional to have a console error here?

Let's render the `<Error />` component instead?

Reply:
Oh, I see. You don't want to stop rendering in the case of an error. I still wonder if it's necessary?

Reply:
Uses the error component instead now. Rebased.